wiki/History_of_osu!/2016/en.md

```diff
@@ -2,125 +2,181 @@
 
 ## January
 
-A brand new year with additions to the list of [community contributors](/wiki/People/Community_Contributors). Best of 2015 voting opened.
+**The ability to see live previews of a selected skin in the options menu was re-added to osu! on 28 January 2016 after having been removed for two years.** The feature was initially removed in 2014 due to an options menu refresh in 2014 [[1,2]][r].
 
-Links:
+## February
 
-- [Community Contributor Badges (January 2016)](https://osu.ppy.sh/home/news/2016-01-09-community-contributor-badges-january-2016)
+**Two new items were added to the osu!store in February: the osu!mug, and the osu!go.** The former being a typical coffee mug with custom osu! designs on it ($15 USD at launch); while the latter was a USB stick that was preloaded with the osu! client, capable of playing osu! directly off of it ($50 USD at launch) [[6,7]][r]. <!--dates missing-->
 
-## February
+![](img/osu-mug-osu-go.jpg "The chibi pippi osu!mug (left) and the osu!go (right) [[6]][r]")
 
-A sneak peek of the osu!next design was on display in the 46th issue of osu!weekly. peppy blogged various improvements for the game. Most notably, the skin preview in the [options](/wiki/Options) and new skinnable interface sounds were added. The [Best of 2015](/wiki/Contests) results were in with [Panda Eyes & Teminite - Highscore (CiiyusMiapah)](https://osu.ppy.sh/beatmapsets/332532#osu/736215), [cillia - Fairytale, (mintong89)](https://osu.ppy.sh/beatmapsets/275331#taiko/624447), [Halozy - Genryuu Kaiko (Yukina)](https://osu.ppy.sh/beatmapsets/253969#fruits/582801), and [D(ABE3) - MANIERA (Jinjin)](https://osu.ppy.sh/beatmapsets/272871#mania/646681) taking in the most votes for the osu!, osu!taiko, osu!catch, and osu!mania respectively. New items were added to the osu!store; such as: the osu!mug (a coffee mug), the osu!mug with pippi (ditto), and osu!go (a USB thumb drive with osu! pre-installed). Thanks to [LiquidPL](https://osu.ppy.sh/users/5044384), reording of the sections, on the new website, in your profile was now possible.
+**On 13 February 2016, a new rule change that attempted to change the then current beatmap difficulty spread criteria in ranked beatmaps sparked massive backlash from the beatmapping community** [[3]][r].
 
-Links:
+While much of the response from the community regarded their disapproval for the new rule change (which attempted to limit the amount of difficulties allowed in a single ranked beatmap), a much larger discussion was started that centred around many users' discontent towards the osu! staff due to what they felt was a lack of transparency and player representation regarding then recent changes in the BAT (Beatmap Appreciation Team) as well as the creation of the RCC (Ranking Criteria Council) [[3]][r].
```

Comment:
use of "regarded" was intentional here; "was" makes it sound like the rest of the sentence is going to talk about the response that was approving of the change

Reply:
right, you could use regard, but not in that way

```suggestion
While much of the community regarded the new rule change in disapproval, as it attempted to limit the amount of difficulties allowed in a single ranked beatmap, a much larger discussion was started that centred around many users' discontent towards the osu! staff due to what they felt was a lack of transparency and player representation regarding then recent changes in the BAT (Beatmap Appreciation Team) as well as the creation of the RCC (Ranking Criteria Council) [[3]][r].
```

Reply:
I realize that if you have to explain a sentence to someone, it probably isn't a good sentence, although I was trying to use "regarded" as in "concern" or "refer to". I'll just use the first suggestion but with parentheses to avoid confusion

```suggestion
While much of the response from the community was in disapproval of the new rule change (which attempted to limit the amount of difficulties allowed in a single ranked beatmap), a much larger discussion was started that centred around many users' discontent towards the osu! staff due to what they felt was a lack of transparency and player representation regarding then recent changes in the BAT (Beatmap Appreciation Team) as well as the creation of the RCC (Ranking Criteria Council) [[3]][r].
```